Search party

abpanic opened the pull request:
# Description

Included change to title search in api file

vanessa-cooper committed:
Search party

abpanic committed:
Update items.py

abpanic committed:
Update items.py

vanessa-cooper commented:
Lovely. Let me take a look.

vanessa-cooper reviewed (changes requested):
Umm… something is not quite right, please take another look and try again.

abpanic committed:
Update items.py

abpanic committed:
Update items.sql

abpanic committed:
updating item.py

updating items.py

abpanic committed:
testing

testing

abpanic committed:
updating search party

update to items.py and items_resource.py
backend\app\dependepcies\items.py
backend\app\api\routes\items\items_resources.py
backend\app\db\repositories\items.py
backend\app\models\schemas\items.py

abpanic committed:
Update items.py

vanessa-cooper commented:
@abpanic I don't want to stress you out, but we really need you to finish this PR :)
